docs/content/concepts/ops-jobs-graphs/jobs.mdx

```diff
@@ -155,18 +160,24 @@ When constructing a job, you can customize how that configuration will be satisf
 
 ### Hardcoded configuration
 
-You can supply a config dictionary. The supplied dictionary will be used to configure the job whenever the job is launched. It will show up in the Dagit Launchpad and can be overridden.
+You can supply a config dictionary or <PyObject object="RunConfig"/> object. The supplied config will be used to configure the job whenever the job is launched. It will show up in the Dagit Launchpad and can be overridden.
```

Comment:
This begs the question of how hard we want to push/promote the use of `RunConfig` over the config dictionary.

e.g. we could say

"You can supply a `RunConfig` (or a legacy config dictionary)."